.github/workflows/flake_update.yaml

```diff
@@ -0,0 +1,34 @@
+name: "flake_update"
+
+on:
+  workflow_dispatch:
+  schedule:
+    # At 7:00am on Monday
+    - cron: "0 7 * * 1"
+
+jobs:
+  flake_update:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install nix
+        uses: cachix/install-nix-action@v25
+        with:
+          install_url: https://releases.nixos.org/nix/nix-2.20.4/install
```

Comment:
Why is the version pinned?

Reply:
So that it only changes deliberately and we can choose to accept updates when we are ready. Otherwise we don't even know which version last worked.

Reply:
Okay, then let's do that for the Github test workflow as well.

Reply:
Good call, done